package.json

```diff
@@ -24,7 +24,7 @@
   "main": "dist/index.js",
   "scripts": {
     "transpile": "trash dist/ && babel src -d dist --copy-files",
-    "prepublishOnly": "npm run transpile",
+    "prepare": "npm run transpile",
     "demo:build": "webpack --mode production --progress",
```

Comment:
Why did we need this change?

Reply:
Because [`prepublishOnly` does not get executed when installing from a GIT package](https://github.com/npm/npm/issues/3055) and [`prepare` hook does](https://github.com/npm/npm/issues/3055#issuecomment-304698029).